src/features/pastelStatistics/__tests__/pastelStatisticsLib.test.ts

```diff
@@ -0,0 +1,45 @@
+import * as pastelStatisticsLib from '../utils'
```

Comment:
Could you rename `pastelStatistics` to `pastel`? Having `pastel` prefix is redundant in features. Please rename every file  and component in this PR that is `pastel` prefixed to have no prefix